plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -430,7 +430,89 @@ test.describe( 'Shopper → Checkout block → Place Order (guest user)', () =>
 	} );
 } );
 
-test.describe( 'Checkout Form Errors (guest user)', () => {
+test.describe( 'Shopper → Place Virtual Order', () => {
+	test.beforeEach( async ( { requestUtils } ) => {
+		await requestUtils.rest( {
+			method: 'PUT',
+			path: 'wc/v3/settings/general/woocommerce_ship_to_countries',
+			data: { value: 'disabled' },
+		} );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
```

Comment:
same as above, this could be `afterAll` instead of `afterEach`

Reply:
I've also adjusted this accordingly, similar to your previous comment.